mica/starcheck/starcheck.py

```diff
@@ -20,60 +22,197 @@
 
 DEFAULT_CONFIG = dict(dbi='sqlite',
                       server=os.path.join(MICA_ARCHIVE, 'starcheck', 'starcheck.db3'),
-                      mp_top_level='/data/mpcrit1/mplogs')
+                      mp_top_level='/data/mpcrit1/mplogs',
+                      timelines_db=dict(dbi='sqlite',
+                                        server=os.path.join(os.environ['SKA'], 'data', 'cmd_states', 'cmd_states.db3')))
 FILES = dict(data_root=os.path.join(MICA_ARCHIVE, 'starcheck'),
              touch_file=os.path.join(MICA_ARCHIVE, 'starcheck', "starcheck_parser.touch"),
              sql_def='starcheck.sql')
 
 # this doesn't belong in starcheck
-def get_timeline_at_date(date):
-    aca_db = Ska.DBI.DBI(dbi='sybase', server='sybase', user='aca_read')
-    return aca_db.fetchone(
+def get_timeline_at_date(date, timelines_db=None):
```

Comment:
mica.utils or should I had some helper methods to either Chandra.cmd_states or a new timelines module?